scripts/update_changelog_and_credits.py

```diff
@@ -178,15 +207,21 @@ def update_changelog(
 
     if 'hotfix' in branch_name:
         previous_release_version = get_previous_release_version(
-            current_release_version)
-        current_version_start = 0
-        previous_version_start = 0
-        for index, line in enumerate(changelog_lines):
-            if 'v%s' % current_release_version in line:
-                current_version_start = index
-            if 'v%s' % previous_release_version in line:
-                previous_version_start = index
-        changelog_lines[current_version_start:previous_version_start] = []
+            'hotfix', current_release_version)
```

Comment:
I thought in a previous PR we added constants for 'hotfix' and 'release'. If so can we use that here and elsewhere?

Reply:
Done.